components/Dialogs/src/MDCAlertContentView.m

```diff
@@ -0,0 +1,59 @@
+/*
+ Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCAlertContentView.h"
+
+@implementation MDCAlertContentView
```

Comment:
If this is a private implementation detail, we tend to create these files under src/private.  It's Objective-c, so we can't enforce any privacy, but hopefully it indicates to our clients that these classes / code are not intended to be directly used, subclasses, etc.